test/fake-intake/.vscode/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
suggestion: I don't think you need this file.

Reply:
As we are in a separate module, in order for vscode to be happy I open the IDE from `test/fake-intake`. This creates ` vscode settings file in `.vscode` because of the extensions I use - notably the freshly installed https://marketplace.visualstudio.com/items?itemName=stuart.unique-window-colors

I can skip the ignore or add this folder ignore to the main `.gitgnore`